tests/ci/run_m1_ec2_instance.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+set -exo pipefail
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# Please run from project root folder!
+# You'll want to set the codebuild env variables set if running locally
+
+# cleanup code
+cleanup() {
+  set +e
+  # delete the various documents that we created
+  for name in ${ssm_document_names};do
+    aws ssm delete-document --name "${name}"
+  done
+}
+
+# we wanna run the cleanup code on exit
+trap cleanup EXIT
+
+# print some information for reference
+echo GitHub PR Number: "${CODEBUILD_WEBHOOK_TRIGGER}"
+echo GitHub Branch Name: "${CODEBUILD_WEBHOOK_HEAD_REF}"
+AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
+echo AWS Account ID: "${AWS_ACCOUNT_ID}"
+echo GitHub Repo Link: "${CODEBUILD_SOURCE_REPO_URL}"
+export CLOUDWATCH_LOG_GROUP="aws-lc-ci-macos-arm-cw-logs"
+
+# get information for ec2 instances
+vpc_id="$(aws ec2 describe-vpcs --filter Name=tag:Name,Values=aws-lc-ci-bm-framework/aws-lc-ci-bm-framework-ec2-vpc --query Vpcs[*].VpcId --output text)"
```

Comment:
should the value for the name tag be `aws-lc-ci-macos-arm/aws-lc-ci-macos-arm-ec2-vpc` which is used [here](https://github.com/awslabs/aws-lc/pull/773/files#diff-8678ff404f71dba6fc0af64009ff973930c1fc8e4f1cda4b9755c381652b6fd3R227)?

Same question for lines 31, 32, and 33.